articles/hll_count_distinct.rst

```diff
@@ -62,7 +62,7 @@ Basically, to be able to observe k leading zeros, we need 2k+1 trials
 k in a data set, HLL concludes that there are approximately 2k+1
 distinct elements.
 
-This is pretty straightforward and simple estimation method. However; it
+This is pretty straightforward and simple estimation method. However, it
```

Comment:
Note that files under `articles/` are verbatim copies of blog posts that management wanted to put here to keep visible longer. There may be more stylistic problems lurking. I haven't changed them beyond rewriting as reStructuredText.